packages/elements/src/components/ino-checkbox/readme.md

```diff
@@ -2,121 +2,6 @@
 
 A checkbox that allows the user to select one or more items from a set of options.
 
-## Usage
-
-The component can be used as follows:
-
-### Web Component
-
-```js
-document
-  .querySelector('ino-checkbox')
-  .addEventListener('checkedChange', (e) =>
-    alert(`The checked state is: ${e.detail}`),
-  );
-```
-
-```html
-<ino-checkbox
-  checked
-  disabled
-  name="<string>"
-  value="<string>"
-  indeterminate
-  id="<string>"
-  tab-index="<number>"
->
-  Label
-</ino-checkbox>
-```
-
-### React
-
-#### Example #1 - Basic
-
-```js
-import { Component } from 'react';
-import { InoCheckbox } from '@inovex.de/elements/dist/react';
-
-class MyComponent extends Component {
-  state = {
-    checked: false,
-  };
-
-  handleCheckboxClick() {
-    this.setState({ checked: !this.state.checked });
-  }
-
-  render() {
-    return (
-      <InoCheckbox
-        value="Apple"
-        onClick={(_) => this.handleCheckboxClick()}
-        checked={this.state.checked}
-      >
-        Apple
-      </InoCheckbox>
-    );
-  }
-}
-```
-
-#### Example #2 - With Types
-
-```js
-import React, { Component } from 'react';
-import { InoCheckbox } from '@inovex.de/elements/dist/react';
-import { Components } from '@inovex.de/elements/dist/types/components';
-
-const Checkbox: React.FunctionComponent<Components.InoCheckboxAttributes> = (
-  props,
-) => {
-  const { value, onClick, checked } = props;
-
-  return (
-    <InoCheckbox value={value} onClick={onClick} checked={checked}>
-      {value}
-    </InoCheckbox>
-  );
-};
-
-class MyComponent extends Component {
-  state = {
-    checked: false,
-  };
-
-  handleCheckboxClick() {
-    this.setState({ checked: !this.state.checked });
-  }
-
-  render() {
-    return (
-      <Checkbox
-        value="Apple"
-        onClick={(_) => this.handleCheckboxClick()}
-        checked={this.state.checked}
-      >
-        Apple
-      </Checkbox>
-    );
-  }
-}
-```
-
-## Additional Hints
-
-By default, the label is always shown on the right side of the checkbox.
-
-**Indeterminate:** The indeterminate status is set by the user. It indicates that a user is indeterminate without changing the checked state. If a checkbox is unchecked and indeterminate then it will lose the indeterminate state on click and change to checked. [Documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Indeterminate_state_checkboxes)
```

Comment:
Are these information or desctiptions now not longer important? I can not find any replacements for this.